Gemfile

```diff
@@ -65,6 +66,7 @@ group(:development, :test) do
   gem "rspec_junit_formatter", "~> 0.2.0"
   gem "simplecov"
   gem "webmock"
+  gem "fauxhai" # for chef-utils gem
```

Comment:
I'll get a fauxhai-ng for you with all the latest platforms up today

Reply:
chef-fauxhai right?

Reply:
It's not our gem so probably just fauxhai-ng

Reply:
for @robbkidd